### PR TITLE
Report ANRs in RUM mobile android SDKs

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do? What is the motivation?

- Adds information on fatal and non-fatal ANR reporting in Datadog, including how to change the default setting for non-fatal ANR reporting

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/real_user_monitoring/error_tracking/mobile/android.md [hunk 1]
```diff
@@ -39,6 +39,31 @@ If you have not set up the Android SDK yet, follow the [in-app setup instruction
 
 For any given error, you can access the file path, line number, and a code snippet for each frame of the related stack trace.
 
+### Report ANRs
+
+An "Application Not Responding" ([ANR][6]) is an Android-specific type of error that gets triggered when the application is unresponsive for too long.
+
+ANRs are only reported through the RUM SDK (not through Logs).
+
+#### Non-fatal ANR reporting
```
Comment (MaelNamNam) [inline content/en/real_user_monitoring/error_tracking/mobile/android.md:48]: Shouldn't we write ANR**s** to remain consistent?

File: content/en/real_user_monitoring/error_tracking/mobile/android.md [hunk 2]
```diff
@@ -39,6 +39,31 @@ If you have not set up the Android SDK yet, follow the [in-app setup instruction
...
+- By default, non-fatal ANRs on *Android 30+* are **disabled** because it would create too much noise over fatal ANRs. However, they non-fatal ANRs are **enabled** on *Android 29 and below* by default because reporting on fatal ANRs is not possible on those versions.
+
+For any Android version, you can override the default setting for reporting non-fatal ANRs by setting `trackNonFatalAnrs` to `true` or `false` when initializing the RUM SDK.
+
+#### Fatal ANRs reporting
+Fatal ANRs are ANRs that result in crashes. They are reported when the application is being unresponsive and leads to the Android OS displaying a popup dialog to the user, who chooses to force quit the app through the popup.
+
+{{< img src="real_user_monitoring/error_tracking/rum-anr-fatal.png" alt="A fatal crash report in Error Tracking." >}}
+
+- In the **Error Tracking** page, fatal ANRs are listed as **individual issues**
```
Comment (MaelNamNam) [inline content/en/real_user_monitoring/error_tracking/mobile/android.md:63]: Maybe we could elaborate on this slightly, saying something like:

- In the **Error Tracking** page, fatal ANRs are grouped based on their similarity, which can result into several **individual issues** being created

**Review by MaelNamNam — CHANGES_REQUESTED** [review]
Very nice, I left a bunch of comments but very minor stuff, thanks a lot!

File: content/en/real_user_monitoring/error_tracking/mobile/android.md (hunk 1 above)
Comment (urseberry) [reply content/en/real_user_monitoring/error_tracking/mobile/android.md:48]: The singular looks better to me. To maintain consistency, the earlier header can say "Report ANR errors"

Comment (MaelNamNam) [reply content/en/real_user_monitoring/error_tracking/mobile/android.md:48]: Whatever you prefer between singular or plural, just need to make it consistent :)

And I'm happy with:
- Report fatal ANR
- Report non-fatal ANR

But I feel that "Report fatal ANR error" is a bit redundant

**Review by 0xnm — COMMENTED** [review]
I agree with suggestions from @MaelNamNam. For the rest it looks good to me.

File: content/en/real_user_monitoring/error_tracking/mobile/android.md (hunk 1 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/error_tracking/mobile/android.md:48]: Will update to Report fatal ANR and Report non-fatal ANR. Thank you both for the input!

**Review by brett0000FF — APPROVED** [review]
Looks good! Just left a few small suggestions.

File: content/en/real_user_monitoring/error_tracking/mobile/android.md [hunk 3]
```diff
@@ -39,6 +39,31 @@ If you have not set up the Android SDK yet, follow the [in-app setup instruction
 
 For any given error, you can access the file path, line number, and a code snippet for each frame of the related stack trace.
 
+### Report ANRs
```
Comment (rtrieu) [inline content/en/real_user_monitoring/error_tracking/mobile/android.md:42]:
```suggestion
### Add ANR reporting
```

**Review by MaelNamNam — APPROVED** [review]
LGTM! Thanks!!

**Pull request merged**